src/System.Runtime/tests/System/Text/StringBuilderTests.netcoreapp.cs

```diff
@@ -199,5 +199,38 @@ public static void Insert_CharSpan_Invalid()
             AssertExtensions.Throws<ArgumentOutOfRangeException>("index", () => builder.Insert(builder.Length + 1, new ReadOnlySpan<char>(new char[0]))); // Index > builder.Length
             AssertExtensions.Throws<ArgumentOutOfRangeException>("requiredLength", () => builder.Insert(builder.Length, new ReadOnlySpan<char>(new char[1]))); // New length > builder.MaxCapacity
         }
+
+        public static IEnumerable<object[]> Equals_String_TestData()
+        {
+            var mediumString = new string('a', 30);
+            var largeString = new string('a', 1000);
+            var extraLargeString = new string('a', 50000); // 40000 is the maximum chunk size
```

Comment:
Looks to me MaxChunkSize = 8000

Reply:
yeah you are right. 
```
        // We want to keep chunk arrays out of large object heap (< 85K bytes ~ 40K chars) to be sure.
        // Making the maximum chunk size big means less allocation code called, but also more waste
        // in unused characters and slower inserts / replaces (since you do need to slide characters over
        // within a buffer).
internal const int MaxChunkSize = 8000;  
```
should I change this this to 8k chars or is this something different